OP2Editor.vcxproj

```diff
@@ -96,7 +104,8 @@ echo regsvr32 exec. time &gt; "$(OutDir)regsvr32.trg" </Command>
       <Outputs>$(OutDir)\regsvr32.trg;%(Outputs)</Outputs>
     </CustomBuildStep>
     <PostBuildEvent>
-      <Command>copy "$(TargetPath)" "C:\Program Files (x86)\OP2Mapper"</Command>
+      <Command>
+      </Command>
```

Comment:
We probably should remove this. Having hard coded paths in an unconditional `copy` command is probably bad practice.

Since `<Command>` is now empty, we should probably remove the tag entirely.